frame/bounties/src/benchmarking.rs

```diff
@@ -91,6 +91,102 @@ fn assert_last_event<T: Config>(generic_event: <T as Config>::Event) {
 	assert_eq!(event, &system_event);
 }
 
+fn setup_subbounty<T: Config>(u: u32, d: u32) -> (
+	T::AccountId,
+	T::AccountId,
+	T::AccountId,
+	BalanceOf<T>,
+	BalanceOf<T>,
+	BalanceOf<T>,
+	BalanceOf<T>,
```

Comment:
create a struct for this kind of object

Reply:
Thanks Shawn, will change it.

Reply:
Done Shawn, moved from tuple to struct object